### ISSUE TITLE
import "TextDecoder" but get "TextEncoder"

### ISSUE THREAD

**Issue by Aitthi** [oven-sh/bun#1813]
### What version of Bun is running?

0.4.0

### What platform is your computer?

Darwin 22.1.0 x86_64 i386

### What steps can reproduce the bug?

```sh
bun init
```
```ts
import { TextDecoder } from 'util'
console.log('TextDecoder', TextDecoder)
```
```sh
❯ bun run index.ts
TextDecoder {
  prototype: TextEncoder {
    encoding: undefined,
    encode: [Function: encode],
    encodeInto: [Function: encodeInto]
  }
}
❯ uname -mprs
Darwin 22.1.0 x86_64 i386
❯ bun -v
0.4.0
```

### What is the expected behavior?

![Screenshot 2566-01-18 at 01 08 18](https://user-images.githubusercontent.com/26670680/212978217-769406b2-3277-45c8-99d9-87101868f668.png)

**Comment by Jarred-Sumner** [comment]
this is fixed in canary 

run 

bun upgrade --canary

**Comment by Electroid** [comment]
Thanks for reporting this! It was fixed in #1716 which will be released in Bun v0.5 (hint: which is shipping later today!)

If you run the following command, the bug should be fixed.

```sh
bun upgrade --canary
```
